merfi/collector.py

```diff
@@ -66,3 +71,36 @@ def debian_release_files(self):
                     if isfile(release_file):
                         result.append(release_file)
         return result
+
+    @property
+    def rpm_repomd_files(self):
```

Comment:
That interface sounds like the best way forward. I'll close this PR and open a new one when I can make time.